resources/assets/js/app.js

```diff
@@ -52,15 +52,33 @@ $.when($.ready).then(() => {
         }
       }); */
 
-  $("#sortable").sortable({
-    stop() {
-      const idsInOrder = $("#sortable").sortable("toArray", {
-        attribute: "data-id",
-      });
+  const sortableEl = document.getElementById("sortable");
+  // eslint-disable-next-line no-undef
+  const sortable = Sortable.create(sortableEl, {
+    disabled: true,
+    animation: 150,
+    forceFallback: !/Windows.+Firefox/.test(navigator.userAgent),
```

Comment:
The fix works now only on windows, MacOS still broken.
However I tested with:

```suggestion
    forceFallback: !(navigator.userAgent.toLowerCase().indexOf("firefox") > -1),
```

And it works also on Mac.